AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -30,6 +30,8 @@ public class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
     
     public lazy var contextManager = ContextManager(delegate: self)
     public lazy var matchMonitor = MatchMonitor(delegate: self)
+    var remoteNotificationManager: RemoteNotificationManager!
```

Comment:
it can be let variable if we create it in `init` of alps manager.

Reply:
If I make it `let`
I get the following errors
`Use of 'self' in method call 'setupAPI' before all stored properties are initialized`
`Use of 'self' in property access 'baseURL' before all stored properties are initialized`

Reply:
Ok, I can make it `let`.  I have removed the delegation